lib/samson/logging.rb

```diff
@@ -46,7 +32,25 @@
   end
 end
 
-# workaround for our server ... always gets SIGHUP even though it does not do file logging
-if ENV["IGNORE_SIGHUP"]
-  Signal.trap(:SIGHUP) {}
+# log 1 message per request to in json/syslog format
+if ENV["RAILS_LOG_WITH_LOGRAGE"]
```

Comment:
Need to add in .env.example with explanation

Reply:
added